ts/torch_handler/base_handler.py

```diff
@@ -33,6 +35,14 @@
             "IPEX is enabled but intel-extension-for-pytorch is not installed. Proceeding without IPEX."
         )
 
+try:
```

Comment:
should we guard it behind some sort of condition? maybe moving it after we found serialized model has .onnx extension?

Reply:
great idea, done